ui/src/app/forms/volume/volume.component.html

```diff
@@ -10,6 +10,7 @@ <h2 *ngIf="volume">Edit volume <i>{{volume.name}}</i></h2>
         </mat-form-field>
         <mat-form-field appearance="outline" class="mid-width">
             <mat-label><span>Size</span></mat-label>
+            <mat-error>Example of valid quantities: 3, 3000M, 3G, 3Gi</mat-error>
```

Comment:
The user may think these 4 quantities are equivalent (especially the "3" without any unit). 
I would give more information on the value, for example:

`300k (300*1000), 30Mi(30*1024²), 3Gi (3*1024³), 3G (3*1000³)`

Reply:
Makes sense. Fixed in [`9177d76` (#7064)](https://github.com/redhat-developer/odo/pull/7064/commits/9177d763ad898f7a3f68b9f3ab57c5b38c5940a5)